merkledag/merkledag.go

```diff
@@ -138,9 +142,22 @@ func (n *dagService) Remove(nd node.Node) error {
 	return n.Blocks.DeleteBlock(nd)
 }
 
+// Create a functtion to get the links for a node, from the node,
```

Comment:
Also go style of docs says that you should start doc comment with the name of the function.